docs/api-reference/contexts.md

```diff
@@ -106,6 +107,16 @@ readonly logger: DBOSLogger
 A reference to DBOS's logger.
 Please see our [logging tutorial](../tutorials/logging.md) for more information.
 
+#### `ctxt.span`
+
+```typescript
+readonly span: Span
+```
+
+An [OpenTelemetry Span](https://opentelemetry.io/docs/concepts/signals/traces/#spans) associated with this function.
+You can assign custom trace attributes to this span.
+Please see our [self-hosting tutorial](../tutorials/self-hosting.md#configuring-otlp-telemetry) for more information.
```

Comment:
I would not link to this page here. I would rather link to the [span API](https://github.com/open-telemetry/opentelemetry-js/blob/f6a075be3204befa970cd4b7d6101419ee5d562e/api/src/trace/span.ts#L32).